### PR TITLE
[CARE-6533] Add Tag page to Bea

### PR DESCRIPTION
**Pull request by fgyimah**
image

### CODE REVIEW

File: app/[localeCode]/tag/[tag]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    searchParams: Record<string, string>;
+}
+
+export async function generateMetadata({ params }: Props): Promise<Metadata> {
+    const { generateUrl } = await routing();
+    const { localeCode, tag } = params;
+
+    return generatePageMetadata({
+        locale: localeCode,
+        title: tag,
```
Comment (yuriyyakym) [inline app/[localeCode]/tag/[tag]/page.tsx:22]: Are tags capitalized? If not, maybe it makes sense to capitalize first letter?

File: modules/InfiniteStories/InfiniteStories.tsx [hunk 2]
```diff
@@ -64,8 +68,8 @@ export function InfiniteStories({
     const locale = useLocale();
     const { load, loading, data, done } = useInfiniteLoading(
         useCallback(
-            (offset) => fetchStories(locale, offset, pageSize, category, excludedStoryUuids),
-            [category, excludedStoryUuids, locale, pageSize],
+            (offset) => fetchStories(locale, offset, pageSize, category, excludedStoryUuids, tag),
```
Comment (yuriyyakym) [inline modules/InfiniteStories/InfiniteStories.tsx:71]: So many arguments already. It probably makes sense to pass these params as an object 🤔

**Review by yuriyyakym — APPROVED** [review]

File: modules/InfiniteStories/InfiniteStories.tsx (hunk 2 above)
Comment (fgyimah) [reply modules/InfiniteStories/InfiniteStories.tsx]: https://github.com/prezly/theme-nextjs-bea/pull/1199/commits/d08d648725706d30a8f48e0e3272e04e2b1ed7a0

File: app/[localeCode]/tag/[tag]/page.tsx (hunk 1 above)
Comment (fgyimah) [reply app/[localeCode]/tag/[tag]/page.tsx]: https://github.com/prezly/theme-nextjs-bea/pull/1199/commits/139b5919829c5d329e2b308e6b9ee33aea18f9c9

**Pull request merged**